### PR TITLE
Add plone 5 profiles

### PR DESCRIPTION
**Pull request by djowett-ftw**
I'm not sure we want to do the last commit (429e24287), but confident of the rest of it.

@jone Btw The idea of adding a `to_plone5` profile (prob without ftw.upgrade) seems more attractive to me today (but I've not done it yet)

### CODE REVIEW

File: ftw/redirector/profiles/uninstall_plone5/registry.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<registry>
+
+    <records prefix="plone.resources/redirector_css"
+             interface='Products.CMFPlone.interfaces.IResourceRegistry'
```
Comment (jone) [inline ftw/redirector/profiles/uninstall_plone5/registry.xml]:
```suggestion
             interface="Products.CMFPlone.interfaces.IResourceRegistry"
```

File: ftw/redirector/profiles/uninstall_plone5/registry.xml [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+             remove="True" />
+
+    <records prefix="plone.bundles/ftw-redirector"
+             interface='Products.CMFPlone.interfaces.IBundleRegistry'
```
Comment (jone) [inline ftw/redirector/profiles/uninstall_plone5/registry.xml]:
```suggestion
             interface="Products.CMFPlone.interfaces.IBundleRegistry"
```

**Review by jone — CHANGES_REQUESTED** [review]
- I'm also not sure about the impact of the last commit; I'm not too familiar yet with the new registry. But no objection if it works 😉 
- Do we need to update classifiers / readme / changelog?

**Comment by djowett-ftw** [comment]
I reverted the last commit as it stops the CSS being minified (de-whitespaced) within `logged-in.css` - and while that doesn't matter much here as there's just 7 rules - it's a good principle for elsewhere.

doc changes made as requested. Plone 5.1 classifier is already there.

will rebase (and squash where appropriate before merge)

**Review by jone — APPROVED** [review]
👍

**Pull request merged**